operator/controllers/common/fake/fakeclient.go

```diff
@@ -2,17 +2,26 @@ package fake
 
 import (
 	lfcv1alpha2 "github.com/keptn/lifecycle-toolkit/operator/apis/lifecycle/v1alpha2"
+	lfcv1alpha1 "github.com/keptn/lifecycle-toolkit/operator/apis/lifecycle/v1alpha1"
+	corev1 "k8s.io/api/core/v1"
+	apiv1 "k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1"
+	utilruntime "k8s.io/apimachinery/pkg/util/runtime"
 	"k8s.io/client-go/kubernetes/scheme"
+	clientgoscheme "k8s.io/client-go/kubernetes/scheme"
 	"sigs.k8s.io/controller-runtime/pkg/client"
 	"sigs.k8s.io/controller-runtime/pkg/client/fake"
 )
 
 // NewClient returns a new controller-runtime fake Client configured with the Operator's scheme, and initialized with objs.
-func NewClient(objs ...client.Object) (client.Client, error) {
-	err := setupScheme()
-	return fake.NewClientBuilder().WithScheme(scheme.Scheme).WithObjects(objs...).Build(), err
+func NewClient(objs ...client.Object) client.Client {
+	setupSchemes()
```

Comment:
why this? Please check and fix linter errors https://github.com/keptn/lifecycle-toolkit/actions/runs/3928484883